types/react-router/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for React Router 4.0
+// Type definitions for React Router 4.1
```

Comment:
I don't think you should increase the version number because that number should only be changed when the type definition needs to reflect a change in the library code. `react-router` is still at 4.0 beta and hasn't reached 4.1 yet.